src/kbmod_wf/resource_configs/klone_configuration.py

```diff
@@ -91,8 +93,8 @@ def klone_resource_config():
                     init_blocks=0,
                     parallelism=1,
                     nodes_per_block=1,
-                    cores_per_node=2,  # perhaps should be 8???
-                    mem_per_node=512,  # In GB
+                    cores_per_node=1,
+                    mem_per_node=128,  # In GB
```

Comment:
Similarly here, we're reducing the number of cores and memory for the kbmod search step.